charts/rd-professional-api/requirements.yaml

```diff
@@ -0,0 +1,7 @@
+dependencies:
+  - name: java
+    version: ~2.0.4
+    repository: '@hmcts'
+  - name: postgresql
```

Comment:
Advise not to use postgresql chart separately and use the one java-chart gives.

Reply:
Done